tests/Microsoft.Identity.Test.Unit.net45/ApiConfigTests/AcquireTokenInteractiveBuilderTests.cs

```diff
@@ -0,0 +1,97 @@
+
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.ApiConfig;
+using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.Identity.Test.Unit.ApiConfigTests.Harnesses;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
+
+namespace Microsoft.Identity.Test.Unit.ApiConfigTests
+{
+    [TestClass]
+    [TestCategory("BuilderTests")]
+    public class AcquireTokenInteractiveBuilderTests
+    {
+        private AcquireTokenInteractiveBuilderHarness _harness;
+
+        [TestInitialize]
+        public async Task TestInitializeAsync()
+        {
+            _harness = new AcquireTokenInteractiveBuilderHarness();
+            await _harness.SetupAsync().ConfigureAwait(false);
+        }
+
+        [TestMethod]
+        public async Task TestAcquireTokenInteractiveBuilderAsync()
+        {
+            await AcquireTokenInteractiveParameterBuilder
+                  .Create(_harness.ClientApplication, MsalTestConstants.Scope, null).ExecuteAsync().ConfigureAwait(false);
+
+            _harness.ValidateCommonParameters(ApiEvent.ApiIds.AcquireTokenWithScope);
+            _harness.ValidateInteractiveParameters();
+        }
+
+        [TestMethod]
+        public async Task TestAcquireTokenInteractiveBuilderWithAccountAsync()
+        {
+            var account = Substitute.For<IAccount>();
+            account.Username.Returns(MsalTestConstants.DisplayableId);
+
+            await AcquireTokenInteractiveParameterBuilder
+                  .Create(_harness.ClientApplication, MsalTestConstants.Scope, null).WithAccount(account).ExecuteAsync().ConfigureAwait(false);
+
+            _harness.ValidateCommonParameters(ApiEvent.ApiIds.AcquireTokenWithScopeUser);
+            _harness.ValidateInteractiveParameters(expectedAccount: account, expectedLoginHint: MsalTestConstants.DisplayableId);
+        }
+
+        [TestMethod]
+        public async Task TestAcquireTokenInteractiveBuilderWithLoginHintAsync()
+        {
+            await AcquireTokenInteractiveParameterBuilder
+                  .Create(_harness.ClientApplication, MsalTestConstants.Scope, null).WithLoginHint(MsalTestConstants.DisplayableId).ExecuteAsync().ConfigureAwait(false);
+
+            _harness.ValidateCommonParameters(ApiEvent.ApiIds.AcquireTokenWithScopeHint);
+            _harness.ValidateInteractiveParameters(expectedLoginHint: MsalTestConstants.DisplayableId);
+        }
+
+        [TestMethod]
+        public async Task TestAcquireTokenInteractiveBuilderWithAccountAndLoginHintAsync()
+        {
+            var account = Substitute.For<IAccount>();
+            account.Username.Returns(MsalTestConstants.DisplayableId);
+
+            await AcquireTokenInteractiveParameterBuilder
+                  .Create(_harness.ClientApplication, MsalTestConstants.Scope, null).WithAccount(account).WithLoginHint("SomeOtherLoginHint").ExecuteAsync().ConfigureAwait(false);
```

Comment:
>.WithAccount(account) [](start = 84, length = 21)

nit/general comment: I believe readability will be better if each "With.." is on new lines. Perhaps even ExecuteAsync. Create can perhaps be in the same line as the await?